content/docs/integrations/grafana/index.md

```diff
@@ -36,12 +36,23 @@ You can choose to configure Grafana in one of two ways:
 
 #### Configure the Grafana endpoints for direct access
 
-Configure the Grafana endpoints to use via `Administration -> Settings -> Grafana`. If you're using grafana.net your
-frontend and backend endpoints should be the same e.g. `https://<your-company>.grafana.net/`. If you're using hosting
-Grafana yourself you'll need to specify a url to the frontend which is used by backstage to generate links and an API
-endpoint which the plugin uses to query alerts and dashboards. 
+Configure the Grafana endpoints to use via `Administration -> Settings -> Grafana`. 
 
-![grafana-config.webp](./grafana-config.webp)
+#### Cloud Hosted & IP Whitelist Restricted
+
+If you're using grafana.net, your frontend and backend endpoints should be the same e.g. `https://<your-company>.grafana.net/`. If you're using hosting Grafana yourself you'll need to specify a url to the frontend which is used by backstage to generate links and an API endpoint which the plugin uses to query alerts and dashboards. 
+
+![grafana-config-cloud](grafana-config-cloud.png)
+
+#### Private Network via Broker
+
+When connecting Grafana from your private network via Broker, you will set the following values for the frontend and backend endpoints.
+
+**Frontend**: URL of your internal frondend (e.g. http://localhost:8888)
+
+**Backend**: `${BROKER_URL}/[BROKER_TOKEN]` (e.g. `${BROKER_URL}/myGrafana`)
```

Comment:
We don't use this syntax on the frontend settings section. Use the broker protocol instead